sphinx/testing/util.py

```diff
@@ -244,3 +240,20 @@ def _clean_up_global_state() -> None:
 
     # clean up autodoc global state
     sphinx.pycode.ModuleAnalyzer.cache.clear()
+
+
+_DEPRECATED_OBJECTS: Final[dict[str, tuple[object, str, tuple[int, int]]]] = {
```

Comment:
@jayaddison this is how we deprecate objects: you only have the name (but not a real object) and you return the "correct" object. See [PEP 562](https://peps.python.org/pep-0562).